### ISSUE TITLE
Icon Request: Gogland

### ISSUE THREAD

**Issue by jags9415** [numixproject/numix-core#3666]
**Website**: https://www.jetbrains.com/go/

```
[Desktop Entry]
Version=1.0
Type=Application
Name=Gogland
Icon=/opt/Gogland/bin/gogland.png
Exec="/opt/Gogland/bin/gogland.sh" %f
Comment=The Drive to Develop
Categories=Development;IDE;
Terminal=false
StartupWMClass=jetbrains-gogland
```

**Original Icon**:
![gogland](https://cloud.githubusercontent.com/assets/3059433/25113414/6ae4b8aa-23b4-11e7-8ea3-4fd347476d3f.png)

**Comment by palob** [comment]
AFAIK this has been renamed and can be closed in favour of #4198, right @jags9415?

**Comment by jags9415** [comment]
That's right @palob, I'm closing this in favour of https://github.com/numixproject/numix-core/issues/4198.
